### DISCUSSION TITLE
How to check for a deleted record in a template

### DISCUSSION THREAD

**Discussion by ColonelBlimp** [aarondl/sqlboiler#1242] [Q&A] [answered]
I am not sure if this is the right place for the question I have, please forgive me if it is not.

I have an small app which can soft delete a record. It also allows the user to 'show deleted'. In the template, I want to be able to 'disable' features for 'deleted' records. I have tried:

` {{ if $e.DeletedAt }}...`

and

`{{ if eq $e.DeletedAt nil }}...`

but none of these seem to work to indicate whether the record is _marked_ as deleted or not.

Can anyone kindly point me in the right direction?

Many thanks.

**Comment by stephenafamo** [comment, marked as answer]
Try `{{ if $e.DeletedAt.Valid }}` to check if it has been deleted

**Reply by ColonelBlimp** [reply]
Outstanding. Many thanks for the quick reply. Appreciated!
